src/common/snippets/src/generator.cpp

```diff
@@ -63,7 +63,9 @@ Generator::opRegType Generator::get_op_reg_type(const std::shared_ptr<Node>& op)
         std::dynamic_pointer_cast<op::LoopBegin>(op) ||
         std::dynamic_pointer_cast<op::LoopEnd>(op) ||
         std::dynamic_pointer_cast<op::Brgemm>(op) ||
-        std::dynamic_pointer_cast<op::Buffer>(op))
+        std::dynamic_pointer_cast<op::Buffer>(op) ||
+        std::dynamic_pointer_cast<op::PerfCountBegin>(op) ||
+        std::dynamic_pointer_cast<op::PerfCountEnd>(op))
```

Comment:
Maybe we should cast to a base class here to cover cpu specific implementation too?

Reply:
great point, done [faf5c65](https://github.com/openvinotoolkit/openvino/pull/19493/commits/faf5c65ed0f9104a67362033488afbd4e6ac5821)